### PR TITLE
Tablet mode toggle

### PR DESCRIPTION
**Pull request by sgober**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:
JIRA 1964 for the tablet toggle between the TDP and POC.  See the JIRA for mockups.

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [x] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [ ] Recognizes any potential shortcomings/bugs in the description 
- [ ] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [x] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [ ] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/dashboard/PointOfCareDashboard.jsx [hunk 1]
```diff
@@ -13,17 +23,98 @@ export default class PointOfCareDashboard extends Component {
...
         };
+        const PanelComponent = this.state.showPOC ? PointOfCarePanel : TargetedDataPanel;
 
         return (
             <div id="point-of-care-dashboard-content" style={{display: "flex"}}>
-                <div className="right-border-box" style={targetedDataPanelStyles}>
-                    <TargetedDataPanel
+                {this.renderSidebar()}
+                <div className="right-border-box" style={panelStyles}>
+                    <PanelComponent
```
Comment (jafeltra) [inline src/dashboard/PointOfCareDashboard.jsx]: This is definitely a question, but the way we have this switch between components set up looks like it means we'll have to pass all the same props into PointOfCarePanel and TargetedDataPanel. I'm not exactly sure everything that the PointOfCarePanel will need to do, but it seems like there could be a chance one component could need props that the other would not. Is it better to just conditionally render the different components? Or is it okay to pass the props and not use them in the components? Or maybe the props will end up being very similar and this is the best way to do it. Just my thoughts that I wanted to mention.

**Comment by jafeltra** [comment]
This looks good! The sidebar looks really cool! A few things I noticed:

[JA1] I think the icons might be swapped. In the mock ups it looks like the lower icon with the 6 boxes is for the POC and the upper icon with the note is for the TDP
[JA2] Do we need that little white arrow that's in the mockups indicating which tab you have selected in the sidebar?
[JA3] The right side of the TDP gets cut off for me sometimes. It happens when I initially load the app, when you switch between POC and TDP, and in tablet portrait mode on the iPad view. It seems like this might be due to the sidebar.
[JA4] This is more of a question, but is there a reason to have the svgs in the public folder if we are adding the svg directly in the component?

File: src/dashboard/PointOfCareDashboard.jsx (hunk 1 above)
Comment (gregquinn2001) [reply src/dashboard/PointOfCareDashboard.jsx]: @jafeltra yes! This is not a planned point of extensibility per se so we should have a state variable that we use to conditionally render either TDP or POC

Comment (sgober) [reply src/dashboard/PointOfCareDashboard.jsx]: @jafeltra @gregquinn2001  you mean have separate render functions for rendering the TDP and POC?  I was thinking about doing that but didn't in case the inputs were similar to the POC  component.  I'm okay either way though, someone make a decision!

**Comment by sgober** [comment]
@jafeltra 

[JA1] you would be correct... fixed
[JA2] not sure... it was not provided with the icons that dan sent. @gregquinn2001 what do you think?
[JA3] I can't recreate this on a normal screen but am seeing this for iPad mode.  looking into it
[JA4] I followed the convention of what we do for other icons.  I can remove them but it seems nice to have them located in the repo.

File: src/dashboard/PointOfCareDashboard.jsx (hunk 1 above)
Comment (gregquinn2001) [reply src/dashboard/PointOfCareDashboard.jsx]: It doesn't need to be separate render functions IMO. I think it should just be a switch statement on the state variables that says whether we are displaying POC or TDP and in the POC case, it returns <PointOfCare.... and in the TDP case it returns <TargetedDataPanel ...

It doesn't need to treat them as the same thing with a common abstract base. They are different other than both being react components. Does that make more sense?

**Comment by gregquinn2001** [comment]
For JA2, I do think we want it there eventually. We can make a separate JIRA for it and ask Dan for it so we can add it in a future sprint

**Comment by gregquinn2001** [comment]
For JA4, we don't use the SVGs at all in the public folder? I'd say don't include them there then? Other thoughts?

**Comment by jafeltra** [comment]
[JA5] The icons look like they're used for the correct display now. But I did notice that in the mockups, they TDP icon is above the POC icon. Also, it looks like in the code, the variables for the `TdpIcons` are actually the icons for the POC and the `PocIcons` variables are the TDP icons. We might want to swa the names of those just so we don't get confused in the future.

**Comment by jafeltra** [comment]
I was also talking to @Dtphelan1 about JA4 and I think we reached the same conclusion as @gregquinn2001 mentioned. I think we should remove the SVGs from the public folder since we don't seem to be using them at all from there.

**Comment by sgober** [comment]
JA3 & JA4 resolved, also addressed the separate render comment

**Comment by jafeltra** [comment]
JA1, JA3, and JA5 all look good to me. JA2 and JA5 are still outstanding I believe.

**Comment by sgober** [comment]
JA5 resolved

File: src/dashboard/PointOfCareDashboard.jsx (hunk 1 above)
Comment (jafeltra) [reply src/dashboard/PointOfCareDashboard.jsx]: The change for this looks good! Thanks!
